pkg/util/bandwidth/linux.go

```diff
@@ -128,15 +128,16 @@ func asciiCIDR(cidr string) (string, error) {
 	return fmt.Sprintf("%s/%d", ip.String(), size), nil
 }
 
-func (t *tcShaper) findCIDRClass(cidr string) (class, handle string, found bool, err error) {
+func (t *tcShaper) findCIDRClass(cidr string) (classAndHandle [][]string, found bool, err error) {
+	classAndHandleList := [][]string{}
```

Comment:
No need to redeclare `classAndHandleList` here.
Just use `classAndHandle` as you declare in the return value.

Reply:
like 
```
 func (t *tcShaper) findCIDRClass(cidr string) (result [][]string,
```

Reply:
Is that it?